main.cc

```diff
@@ -552,6 +552,16 @@ static locator::host_id initialize_local_info_thread(sharded<db::system_keyspace
     return linfo.host_id;
 }
```

Comment:
> (cherry picked from commit https://github.com/scylladb/scylladb/commit/a7096ebb28d06161599e8acc0641b3870713d023)

Doesn't refer to a commit in the repository.

Reply:
Why are we doing PGO stuff here?

Reply:
Coverage reports and PGO use the same profiles/instrumentation.

Reply:
yes but the commit message is misleading.

Reply:
Because the commit was cherry-picked as-is from the PGO series.

Reply:
I have reworded the commit message.
@avi should I loose the cherry-pick link too?

Reply:
yes, we don't want dangling references

Reply:
reference removed :slightly_smiling_face: